Array/src/ArrayTask4.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class ArrayTask4 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String userAction;
```

Comment:
String variables could be declarated in one line (like String userAction, enterWords etc)

Reply:
Done